Application/Client/src/app/createCard/component/createCard.component.ts

```diff
@@ -191,7 +196,7 @@ export class CreateCardComponent implements OnInit {
 
     isAddInfoCompleted() {
         console.log(this.myForm)
-        if(this.myForm.get('title').value!='' && this.myForm.get('price').value!='' && this.myForm.get('shortDesc').value!='' && this.myForm.get('negotiable').value!='' && this.myForm.get('location').value!=''){
+        if(this.myForm.get('title').value!='' && this.myForm.get('price').value!='' && this.myForm.get('shortDesc').value!='' && this.myForm.get('location').value!=''){
```

Comment:
use strict operator !== and give some space into condition it will help to understand e.g this.myForm.get('title').value !== ''